From grinberg@compulab.co.il  Wed Jul 21 13:08:19 2010
From: Igor Grinberg <grinberg@compulab.co.il>
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: linux-usb@vger.kernel.org, linux-arm-kernel@lists.infradead.org,
	Sascha Hauer <kernel@pengutronix.de>, Daniel Mack <daniel@caiaq.de>,
	=?UTF-8?q?Eric=20B=E9nard?= <eric@eukrea.com>,
	Mike Rapoport <mike@compulab.co.il>,
	Igor Grinberg <grinberg@compulab.co.il>
Subject: USB: otg/ulpi: remove unused macro
Date: Thu, 15 Jul 2010 16:00:14 +0300
Message-Id: <1279198816-30367-2-git-send-email-grinberg@compulab.co.il>


Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
Signed-off-by: Mike Rapoport <mike@compulab.co.il>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/otg/ulpi.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/usb/otg/ulpi.c
+++ b/drivers/usb/otg/ulpi.c
@@ -31,8 +31,6 @@
 
 #define ULPI_ID(vendor, product) (((vendor) << 16) | (product))
 
-#define TR_FLAG(flags, a, b)	(((flags) & a) ? b : 0)
-
 /* ULPI hardcoded IDs, used for probing */
 static unsigned int ulpi_ids[] = {
 	ULPI_ID(0x04cc, 0x1504),	/* NXP ISP1504 */
